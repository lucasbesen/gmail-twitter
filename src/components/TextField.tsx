import styled from 'styled-components';

const TextField = styled.input`
  background-color: #eee;
  font-size: 15px;
  padding: 15px;
  border-radius: 8px;
  width: 50%;
  border: 0;
  outline: 0;

  :focus {
    background-color: #ffffff;
    box-shadow: 0px 1px 3px 0px #d3d3d3;
  }
`;

export default TextField;
