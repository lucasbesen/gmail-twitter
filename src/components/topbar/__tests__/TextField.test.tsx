import React from 'react';
import renderer from 'react-test-renderer';

import TextField from '../TextField';

it('should render TextField without error', () => {
  const component = renderer.create(<TextField />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
