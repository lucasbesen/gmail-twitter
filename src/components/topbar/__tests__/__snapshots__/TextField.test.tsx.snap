// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TextField without error 1`] = `
<input
  className="sc-bdVaJa bGmHtF"
/>
`;
